lib/cpp/robust/model_absolute_regression.cpp

```diff
@@ -24,3 +25,7 @@ T TModelAbsoluteRegression<T>::grad_i_factor(const ulong i,
 
 template class DLL_PUBLIC TModelAbsoluteRegression<double>;
 template class DLL_PUBLIC TModelAbsoluteRegression<float>;
+
+// template class DLL_PUBLIC TModelAbsoluteRegression<double,
+// std::atomic<double>>; template class DLL_PUBLIC
+// TModelAbsoluteRegression<float, std::atomic<float>>;
```

Comment:
forgot this?

Reply:
Or is this deactivated on purpose for all robust models?

Reply:
should be active